packages/apm/src/span.ts

```diff
@@ -153,6 +153,19 @@ export class Span implements SpanInterface, SpanContext {
     return span;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public getTransaction(): Transaction {
+    const recorder = this.spanRecorder;
+    if (!recorder) {
+      logger.warn('This Span has no reference to a Transaction. Returning an instance to itself.');
+      logger.warn('It means that the Transaction has been sampled or the Span did not originate from a Transaction.');
+      return (this as unknown) as Transaction;
+    }
+    return recorder.spans[0] as Transaction;
```

Comment:
I guess it would be more robust to handle the case where `recorder` is truthy but `recorder.spans` is empty. Does that make sense here?